service/src/main/java/edu/tamu/catalog/controller/CatalogAccessController.java

```diff
@@ -64,6 +68,24 @@ public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="e
         }
     }
 
+    /**
+     * Provides data for all fees and fines associated with a patron.
+     *
+     * @param String catalogName (optional)
+     * @param String user
+     * @return
+     */
+    @GetMapping("/fines")
+    public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="folio") String catalogName, @WeaverUser User user) {
```

Comment:
As described in slack, we will not use WeaverUser here nor have it require authorization. The Catalog service is an interface between other services that will provide authorization. By the time a request gets to Catalog Service it should have already been authorized. I would also think we should implement a white list for access to the Catalog service.